src/app/auth/login/login.component.scss

```diff
@@ -127,6 +129,11 @@
                 &:hover {
                     background-color: $secondary;
                 }
+
+                &:disabled {
+                    background-color: #757575;
```

Comment:
Kindly assigned this color to a variable and make it reusable it. It has been used at other place